redis/typing.py

```diff
@@ -32,7 +32,14 @@
 PatternT = _StringLikeT  # Patterns matched against keys, fields etc
 FieldT = EncodableT  # Fields within hash tables, streams and geo commands
 KeysT = Union[KeyT, Iterable[KeyT]]
-ResponseT = Union[Awaitable[Any], Any]
+OldResponseT = Union[Awaitable[Any], Any]  # Deprecated
+AnyResponseT = TypeVar("AnyResponseT", bound=Any)
+ResponseT = Union[AnyResponseT, Awaitable[AnyResponseT]]
+OKT = Literal[True]
+ArrayResponseT = List
+IntegerResponseT = int
+NullResponseT = type(None)
+BulkStringResponseT = str
```

Comment:
Again, why not use `str` directly?

Reply:
for clarity, it isn't just any `str` it is a `BulkStringResponseT` even better would be a uage of newtype https://docs.python.org/3/library/typing.html#newtype